src/coreclr/gc/gc.cpp

```diff
@@ -8359,29 +8427,197 @@ void destroy_card_table (uint32_t* c_table)
     dprintf (2, ("Table Virtual Free : %Ix", (size_t)&card_table_refcount(c_table)));
 }
 
-uint32_t* gc_heap::make_card_table (uint8_t* start, uint8_t* end)
+void gc_heap::get_card_table_element_sizes (uint8_t* start, uint8_t* end, size_t card_table_element_sizes[total_bookkeeping_elements])
```

Comment:
would be good to not use the same name `card_table_element_sizes` as the global var.